packages/touchpoint-ui/src/voice.ts

```diff
@@ -56,79 +124,80 @@ export const useVoice = ({
     };
   }, [disconnect]);
 
+  const setupRoom = useCallback(async (): Promise<void> => {
+    try {
+      setRoomState("pending");
+
+      const creds = await handler.getLiveKitCredentials();
+
+      const handleActiveSpeakersChanged = (
+        participants: Participant[],
+      ): void => {
+        const hasAgent = participants.some(
+          (participant) => participant.isAgent,
+        );
+        const hasLocal = participants.some(
+          (participant) => participant.isLocal,
+        );
+        setIsApplicationSpeaking(hasAgent);
+        setIsUserSpeaking(hasLocal);
+      };
+
+      const handleTrackSubscribed = (track: RemoteTrack): void => {
+        if (track.kind === Track.Kind.Audio) {
+          const element = track.attach();
+          element.play().catch((err) => {
+            // eslint-disable-next-line no-console
+            console.warn(err);
+          });
```

Comment:
```suggestion
          void element.play();
```

should work too, no?